packages/clerk-js/src/core/fapiClient.ts

```diff
@@ -199,13 +199,16 @@ export default function createFapiClient(clerkInstance: Clerk): FapiClient {
 
     try {
       if (beforeRequestCallbacksResult) {
+        const maxTries = isBrowserOnline() ? 4 : 11;
```

Comment:
This check here determines the maxTries the moment the request begins.
If we move it inside shouldRetry, it will also catch cases where the browser goes offline in the middle of the request. Which one do you prefer @desiprisg ?

Reply:
I don't think that's better. In that case, if the browser's offline and then becomes online and we're at the 6th try, the check will fail.